requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+flake8
```

Comment:
All the dependencies are listed in these two files `requirements-dev.txt` and `requirements.txt`.
This is easier to maintain and they get propagated to whatever package manager is used later.

PS: Note that I moved the code to a main `seapy` module folder.
Only the packaging metadata lives in the root folder now.